web3j/src/main/java/com/vmware/ethereum/config/AppConfig.java

```diff
@@ -103,6 +115,52 @@ public Credentials credentials() throws IOException, CipherException {
         credentials.getWalletPassword(), credentials.getWalletFile());
   }
 
+  @Bean
+  public TransactionReceiptProcessor queuedTransactionReceiptProcessor(
+      Web3j web3j, CountDownLatch countDownLatch, MetricsService metrics) {
```

Comment:
What is the purpose of passing countDownLatch and metrics if you are not using it?

Reply:
It is used in callback() function , removed it from transactionManager where it was not used